### PR TITLE
Add configurable MAPSERVER_BASE_PATH

### PR DESCRIPTION
**Pull request by danduk82**
- [x] fix base path not working anymore
- [x] add readme entry
- [x] add tests for capabilities

### CODE REVIEW

**Review by sbrunner — APPROVED** [review]
Just some small comments, thanks :-)

**Comment by sbrunner** [comment]
Yon can install the pre-commit hook: https://github.com/camptocamp/docker-mapserver#contributing :-)

File: Makefile [hunk 1]
```diff
@@ -22,8 +22,12 @@ all: acceptance
 build: ## Build the Docker image
 	docker $(DOCKER_BUILDX) build $(DOCKER_BUILD_ARGS) --tag=$(DOCKER_IMAGE):$(DOCKER_TAG) --target=runner --build-arg=MAPSERVER_BRANCH=$(MAPSERVER_BRANCH) --build-arg=WITH_ORACLE=$(WITH_ORACLE) .
 
+.PHONY: build-no-cache
+build-no-cache: ## Build the Docker image (no cache)
+	docker $(DOCKER_BUILDX) build --no-cache $(DOCKER_BUILD_ARGS) --tag=$(DOCKER_IMAGE):$(DOCKER_TAG) --target=runner --build-arg=MAPSERVER_BRANCH=$(MAPSERVER_BRANCH) --build-arg=WITH_ORACLE=$(WITH_ORACLE) .
+
 .PHONY: acceptance
-acceptance: build ## Run the acceptance tests
+acceptance: # build ## Run the acceptance tests
```
[the remark this answers is not in the document]
Comment (danduk82) [reply Makefile:30]: oups

[the remark this answers is not in the document]
Comment (danduk82) [reply Makefile]: done

**Comment by shayneoneill** [comment]
Hey, whats supposed to go in this env variable? The synology docker runner explodes cos theres an empty variable here, but I dont know what I'm supposed to feed it?

**Comment by sbrunner** [comment]
If you don't need it, keep it empty :-)

**Pull request merged**